src/UserManager.ts

```diff
@@ -638,11 +635,20 @@ export class UserManager {
             throw err;
         }
     }
+
     protected async _signoutEnd(url: string): Promise<SignoutResponse> {
         const logger = this._logger.create("_signoutEnd");
         const signoutResponse = await this._client.processSignoutResponse(url);
         logger.debug("got signout response");
 
+        try {
```

Comment:
please remove this try catch, if `this.removeUser()` there is something really wrong

Reply:
Done :slightly_smiling_face: